rgw/v2/tests/s3_swift/test_bucket_lc_object_exp_multipart.py

```diff
@@ -149,7 +149,7 @@ def test_exec(config, ssh_con):
 
                 if config.local_file_delete:
                     log.info("deleting local file created after the upload")
-                    utils.exec_shell_cmd(f"rm -rf {TEST_DATA_PATH}")
+                    utils.exec_shell_cmd(f"rm -rf {TEST_DATA_PATH}/*")
```

Comment:
Please delete particular object using `s3_object_path` instead of deleting all objects present in `test_data` directory to avoid failures which might occure in future while doing parallel execution

Reply:
@udaysk23, With s3_object_path dint work went ahead with different approach

Reply:
Updated same log